### ISSUE TITLE
When missing credentials the error should specify for the service name

### ISSUE THREAD

**Issue by germanattanasio** [watson-developer-cloud/node-sdk#357]
When using multiple services is pretty hard to know which service is missing credentials. The error is very generic.
```
Error: Argument error: api_key or username and password were not specified
```

Maybe we can do something like:

```
Error: Argument error: api_key or username and password were not specified for Speech to Text v3
```

**Comment by nfriedly** [comment]
The data is always there in the stack trace (`Error.stack`), but putting it into the message isn't a bad idea.

**Comment by adrien2p** [comment]
hi every body, i've opened a pull request that add this feature.

good night.

**Comment by ammardodin** [comment]
I will make sure to include this change in the upcoming release.

**Comment by ammardodin** [comment]
This is included in the beta release soon to be 3.0.0.
